shacl/index.html

```diff
@@ -491,14 +491,14 @@
 				</p>
 				<p>
 					Note that <code>rdfs:subClassOf</code> transitivity is only used when SHACL explicitly determines type and subclass relationships in the shapes graph and the data.
-					In other places only triples that are explicitly present in the shapes graph or the data are considered.
+					Elsewhere, only triples that are explicitly present in the shapes graph or the data are considered.
 					For example, SHACL property constraints on <code>rdf:type</code> and <code>rdfs:subClassOf</code> only utilize triples that are explicitly in the data.
 					Similarly, subproperties of <code>rdfs:label</code> and <code>rdfs:commment</code> and subclasses of <code>rdf:List</code> are not recognized in the shapes graph.
 				</p>
 				<p>
 					These design decisions mean that SHACL processors do not have to natively support full RDFS inferencing.
-					However, SHACL validation engines may operate on RDF graphs that include entailments - either pre-computed before being submitted to a SHACL processor or performed on the fly as part of SHACL processing.
-					To support on the fly processing of entailments, SHACL includes the property <a href="#entailment"><code>sh:entailment</code></a> to indicate what kind of inferencing is required by a given shapes graph.
+					However, SHACL validation applications may operate on RDF graphs that include entailments - either pre-computed before being submitted to a SHACL processor or performed on the fly as part of SHACL processing.
```

Comment:
Right, I meant just "implementations" if it's a statement applying to all use cases (or "SHACL implementations" but I'm not sure that adds anything). And "validation engines" if it's specific to implementations doing validation.